svg.go

```diff
@@ -89,9 +88,29 @@ type DrawableSVG struct {
 
 // SVG holds an SVG.
 type SVG struct {
-	unscaledPath  *Path
-	scaledPathMap map[Size]*Path
-	size          Size
+	paths               []*Path
+	combinedPath        *Path
+	scaledCombinedPaths map[Size]*Path
+	size                Size
+	parseErrorMode      svg.ErrorMode
+	ignoreUnsupported   bool
+}
+
+// SVGOption is an option that may be passed to SVG construction functions.
+type SVGOption func(s *SVG)
+
+// SVGOptionIgnoreParseErrors is an option that will ignore some errors when parsing an SVG.
+// If the XML is not well formed an error will still be generated.
+func SVGOptionIgnoreParseErrors(s *SVG) {
+	s.parseErrorMode = svg.IgnoreErrorMode
```

Comment:
It might be better for this to be `svg.WarnErrorMode` so that issues can be noticed... or maybe just another option for that.